api/structure/schema.py

```diff
@@ -0,0 +1,40 @@
+import graphene
+from graphql import GraphQLError
+from graphene_sqlalchemy import SQLAlchemyObjectType
+from sqlalchemy import func
+from helpers.auth.authentication import Auth
+from api.structure.models import Structure as StructureModel
+
+
+class Structure(SQLAlchemyObjectType):
+    class Meta:
+        model = StructureModel
+
+
+class Query(graphene.ObjectType):
+    """
+      Query for office structures
+    """
+    all_structures = graphene.List(Structure)
```

Comment:
hey @rajeman, Great work so far. Please add the descriptions of the different queries. if help is needed, then reach out to me